src/lithopscloud/modules/cloud_functions/cloud_function.py

```diff
@@ -0,0 +1,147 @@
+import http.client
+import json
+import sys
+from typing import Dict, Any
+import requests
+from lithopscloud.modules.config_builder import ConfigBuilder, spinner
+from lithopscloud.modules.utils import get_option_from_list_alt, free_dialog, color_msg, Color
+
+CF_REGIONS = ['eu-de', 'eu-gb', 'us-south', 'us-east', 'au-syd', 'jp-tok']
+
+
+class CloudFunction(ConfigBuilder):
+
+    def __init__(self, base_config: Dict[str, Any]) -> None:
+        super().__init__(base_config)
+
+    def run(self, api_key=None) -> Dict[str, Any]:
+
+        is_cf_based_namespace = False  # set to true if namespace is cloud foundry based
+        print(color_msg("\n\nConfiguring IBM cloud functions:\n",color=Color.YELLOW))
+
+        choices = CF_REGIONS[:]
+
+        choices.insert(0, "Search across all regions. -*- Will take considerably longer to attain -*-")
+        selected_region = get_option_from_list_alt("Choose a region where your preferred namespace can be found,"
+                                                   " or created in", choices)['answer']
+
+        if 'Present all existing namespaces' in selected_region:
```

Comment:
Present -> List

Reply:
agreed.

Reply:
Actually you pointed out a real issue, it should search for the string 'Search across all regions' instead. The string containing "present" belongs to an older commit.